
<!-- RIGHT HAND NAVIGATION BOX INCLUDE -->
<?php
include('http://www.uvm.edu/~cems/cs/newsevents/_newseventssuppnav.html');
?>

<h3>Seminar Description</h3>

<h4 class="subtitle">"Very Large Scale ReliefF for Genome-Wide Association Analysis"</h4>

<p><a href="http://www.cs.uvm.edu/%7Eeppstein/"><strong>Dr. Maggie Eppstein</strong></a><br />
Department of Computer Science, University of Vermont</p>

<p><strong>October 13, 2008</strong><br>
12:20 - 1:15 pm<br>
367 Votey Hall</p>

<p><strong>Summary</strong><br>
Most common diseases are the result of complex nonlinear interactions
between multiple genetic and environmental components. There is
thus a pressing need for new computational methods capable of detecting
nonlinearly interacting single nucleotide polymorphism (SNPs) that are
associated with disease, from amidst up to hundreds of thousands of
candidate SNPs. Recently, some progress has been made using
feature selection algorithms based on weights from the ReliefF data
mining algorithm on sets of up to 1500 SNPs. However, the accuracy of
ReliefF does not scale up to the sizes needed for truly large
genome-scale SNP association studies. We propose a population-based
variant dubbed VLSReliefF, which mitigates this performance drop by
stochastically applying ReliefF to SNP subsets, and then assigning each
SNP the maximum ReliefF weight it achieved in any subset. A heuristic
method is proposed for determining the optimal subset size as a
function of heritability, sample size, and order of interactions.
Aggressive iterative pruning of SNPs with low VLSReliefF weights can be
used for nonlinear feature identification in genome scale SNP sets. The
method is validated using a variety of computational experiments on
synthetic datasets of up to 100,000 SNPs. While this algorithm
was developed in the context of SNP association problem, the approach
itself is very general and can be applied to many problems that require
high-dimensionality association analysis.
</p>

<p><strong>Speaker Bio</strong>
<br>
Dr. Eppstein is currently an associate professor in Computer Science
and the Director of the Complex Systems Center at the University of
Vermont. She has interdisciplinary background and interests, with
a B.S. in Zoology from Michigan State University, an M.S. in Computer
Science from the University of Vermont, and a Ph.D. in
environmental engineering at the University of Vermont. Her
research interests involve computational modeling and analysis of
complex systems in a variety of domains, both theoretical and
applied. Current projects include developing, studying, and using
novel evolutionary and agent-based computational approaches for a wide
range of important problems, including plant species&#8217; invasiveness in
ecological communities, biological speciation, the impact of spatial
topologies on information flow through complex interaction networks,
identifying nonlinear interactions between single nucleotide
polymorphisms that predispose for complex disease traits, and
agent-based integrated assessment modeling of transportation energy
alternatives.
</p>

<p><span class="return"><a href="../../?Page=default.php">CS homepage</a></span>
| <span class="return"><a href="../../?Page=newsevents/seminars.php">Seminars
page</a></span></p>
