<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 9">
<meta name=Originator content="Microsoft Word 9">
<link rel=File-List href="./Zucker_files/filelist.xml">
<title>Considering Multiple Factors </title>
<!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Byung-Suk Lee</o:Author>
  <o:LastAuthor>Byung-Suk Lee</o:LastAuthor>
  <o:Revision>3</o:Revision>
  <o:TotalTime>10</o:TotalTime>
  <o:Created>2002-10-01T19:31:00Z</o:Created>
  <o:LastSaved>2002-10-01T19:31:00Z</o:LastSaved>
  <o:Pages>1</o:Pages>
  <o:Words>310</o:Words>
  <o:Characters>1771</o:Characters>
  <o:Company>University of Vermont</o:Company>
  <o:Lines>14</o:Lines>
  <o:Paragraphs>3</o:Paragraphs>
  <o:CharactersWithSpaces>2174</o:CharactersWithSpaces>
  <o:Version>9.2720</o:Version>
 </o:DocumentProperties>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:Zoom>BestFit</w:Zoom>
 </w:WordDocument>
</xml><![endif]-->
<style>
<!--
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-parent:"";
	margin:0pt;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	mso-bidi-font-size:10.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";}
@page Section1
	{size:612.0pt 792.0pt;
	margin:72.0pt 90.0pt 72.0pt 90.0pt;
	mso-header-margin:36.0pt;
	mso-footer-margin:36.0pt;
	mso-paper-source:0;}
div.Section1
	{page:Section1;}
-->
</style>
<!--[if gte mso 9]><xml>
 <o:shapedefaults v:ext="edit" spidmax="1026"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <o:shapelayout v:ext="edit">
  <o:idmap v:ext="edit" data="1"/>
 </o:shapelayout></xml><![endif]-->
</head>

<body bgcolor="#f8f8f8" background="./Zucker_files/image001.jpg" lang=EN-US
style='tab-interval:36.0pt'>
<!--[if gte mso 9]><xml>
 <v:background id="_x0000_s1025" o:bwmode="white" fillcolor="#f8f8f8">
  <v:fill src="./Zucker_files/image001.jpg" o:title="Newsprint" type="tile"/>
 </v:background></xml><![endif]-->

<div class=Section1>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt;mso-bidi-font-size:10.0pt'>Algorithms and Statistics
for Nucleic Acid<o:p></o:p></span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt;mso-bidi-font-size:10.0pt'>Secondary Structure
Prediction<o:p></o:p></span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'>Dr. Michael Zuker<o:p></o:p></span></b></p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'>Department of Mathematical
Sciences<o:p></o:p></span></p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'>Rensselaer Polytechnic
Institute<o:p></o:p></span></p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'>Troy, NY 12180<o:p></o:p></span></p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></span></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'>Date</span></b><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'>: Wednesday, October 16,
2002<o:p></o:p></span></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'>Time</span></b><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'>: 12:20 p.m. - 1:20 p.m. <o:p></o:p></span></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'>Location</span></b><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'>: Given Hall B4<o:p></o:p></span></p>

<p class=MsoNormal><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></p>

<p class=MsoNormal><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:14.0pt;mso-bidi-font-size:10.0pt'>Abstract<o:p></o:p></span></b></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:12.0pt;
mso-bidi-font-size:10.0pt'><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></span></b></p>

<p class=MsoNormal style='margin-top:6.0pt;text-align:justify'><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'>Secondary structures
(foldings) for single-stranded nucleic acids can be predicted for single
sequences, or for groups of homologous sequences that can be reliably aligned.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-top:6.0pt;text-align:justify'><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'>The foldings of individual
sequences are predicted using dynamic programming methods borrowed and adapted
from algorithms used to align molecular sequences. Prediction is based on the
minimization of free energy, and nearest neighbor parameters derived from
physical chemistry are used to assign these energies.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-top:6.0pt;text-align:justify'><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'>Minimum energy folding works
very well on some sequences, poorly on others, and within a given sequence is
often more reliable in some regions than in others. The prediction of multiple
secondary structures close to the minimum predicted energy mitigates the uncertainty.
Dot plots that display all possible foldings close to the minimum energy or
base pair probabilities also help. These dot plots convey ``well-definedness''
information that may be mapped onto individual foldings to show what regions
are better predicted than others.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-top:6.0pt;text-align:justify'><span
style='font-size:12.0pt;mso-bidi-font-size:10.0pt'>When a number of aligned,
homologous sequences are available, mutual information computations allow one
to determine base pairs that are conserved in evolution even though the
sequences themselves vary. Phylogenetic information and energy methods may be
used together, each helping the other, when only a few aligned sequences are
available. Large databases of aligned RNAs now exist where secondary structure
has been predicted with confidence. Statistical analysis of the frequencies of small
structural motifs can lead to folding rules that are analogous to the energy
rules currently used. Such studies may eventually lead to independent,
statistically derived rules. Even if this is not achieved, the statistical
folding parameters can point out significant motifs that can then be
investigated by physical chemists or structural biologists.</span></p>

</div>

</body>

</html>
